image_explorer/public/css/image_explorer.css

```diff
@@ -51,6 +51,12 @@
     transition: all 0.0s linear 0s;
 }
 
+
+.image-explorer-hotspot.latest{
+    margin-left: -20.5px;
+    margin-top: -20.5px;
```

Comment:
Can we add a comment to explain that this is half the width/height of the hotspot image?